imsc1/ttml/linePadding/LinePadding006.ttml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tt:tt xmlns:tt="http://www.w3.org/ns/ttml"
+    xmlns:ebutts="urn:ebu:tt:style"
+    xmlns="http://www.w3.org/ns/ttml" 
+    xmlns:tts="http://www.w3.org/ns/ttml#styling" 
+    ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1/text"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter"
+    xml:lang="en">
+    <head>
+        <styling>
+            <style xml:id="paragraphStyle" tts:textAlign="start" tts:lineHeight="135%" ebutts:linePadding="0.2c" />
```

Comment:
Using the value `0.2c` makes it hard to distinguish the case where linePadding is applied from that where it is not, because the value is quite small. Suggest increasing to something really obvious like `1c`, or including two paragraphs, one with the `ebutts:linePadding` set and one where it is not, with the text otherwise the same, so the differences are observable.

Reply:
Addressed.